crates/wallets/Cargo.toml

```diff
@@ -47,9 +47,19 @@ thiserror.workspace = true
 tracing.workspace = true
 eth-keystore = "0.5.0"
 
+# browser wallet dependencies
+axum = { version = "0.7", optional = true }
+tokio = { workspace = true, features = ["net", "rt-multi-thread"], optional = true }
+uuid = { version = "1.10", features = ["v4"], optional = true }
+hex = { version = "0.4", optional = true }
+parking_lot = { version = "0.12", optional = true }
+serde_json = { version = "1.0", optional = true }
+
 [dev-dependencies]
-tokio = { workspace = true, features = ["macros"] }
+tokio = { workspace = true, features = ["macros", "time"] }
+reqwest = { version = "0.12", features = ["json"] }
```

Comment:
maybe this a lot of bloat and we should consider introducing a dedicated browser-wallet crate for this